docs/docusaurus/docs/cloud/connect/connect_postgresql.md

```diff
@@ -0,0 +1,186 @@
+---
+sidebar_label: 'Connect GX Cloud to PostgreSQL'
+title: 'Connect GX Cloud to PostgreSQL'
+description: Connect GX Cloud to a PostgreSQL Data Source.
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To validate data stored in a PostgreSQL database from GX Cloud, you must add the GX Agent to your deployment environment. The GX Agent acts as an intermediary between GX Cloud and PostgreSQL and allows you to securely access and validate your data in GX Cloud.
+
+New to GX Cloud and not sure that it's the right solution for your organization? See [Try GX Cloud](../try_gx_cloud.md).
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud) with [Admin or Editor permissions](../about_gx.md#roles-and-responsibilities).
+
+- You have a PostgreSQL database, schema, and table.
+
+- To improve data security, GX recommends creating a separate PostgreSQL user for your GX Cloud connection.
+
+- You know your PostgreSQL access credentials.
+
+- You have stopped all local running instances of the GX Agent.
+
+## Prepare your PostgreSQL environment
+
+Run the following code to create and assign the `gx_ro` role and allow GX Cloud to access to all `public` schemas and tables on a specific database:
+
+```sql
+   CREATE ROLE gx_ro WITH LOGIN PASSWORD <your_password>;
+   GRANT CONNECT ON DATABASE <your_database> TO gx_ro;
+   GRANT USAGE ON SCHEMA public TO gx_ro;
+   GRANT SELECT ON ALL TABLES in SCHEMA public TO gx_ro;
+```
+
+Replace `your_password` and `your_database` with your own values.
+
+## Get your GX Cloud access token and organization ID
+
+You'll need your access token and organization ID to set your access credentials. Don't commit your access credentials to your version control software.
+
+1. In GX Cloud, click **Settings** > **Tokens**.
+
+2. In the **User access tokens** pane, click **Create user access token**.
+
+3. In the **Token name** field, enter a name for the token that will help you quickly identify it.
+
+4. Click **Create**.
+
+5. Copy and then paste the user access token into a temporary file. The token can't be retrieved after you close the dialog.
+
+6. Click **Close**.
+
+7. Copy the value in the **Organization ID** field into the temporary file with your access token and then save the file. 
+
+   GX recommends deleting the temporary file after you set the environment variables.
+
+## Deploy the GX Agent
+
+You deploy and run the GX Agent within your deployment environment. You can deploy the GX Agent container in any environment where you can run Docker container images or create Kubernetes clusters.
```

Comment:
```suggestion
You deploy and run the GX Agent within your deployment environment. You can deploy the GX Agent container in any environment where you can run Docker container images.
```

Containers are what run inside of a Kubernetes cluster. If someone sees a helm chart or Kubernetes manifest the only place they can deploy that is a Kubernetes cluster. The phrasing just seems kind of off since Kubernetes is at a way higher level up the stack than a container.